Fixed email sending

MeridianExplorer opened the pull request:
When sending messages, as part of the header creation, pyOutlook would try to access 'self.account.access_token'. 
Upon debugging, account actually doesn't have an 'access_token' attribute, and 'account' itself is a string containing the access token.
Upon removal of the redundant 'access_token', sending messages is now possible again.
Tested by me.

JensAstrup commented:
`OutlookAccount` does have an `access_token` attribute. The line you edited though should be using `OutlookAccount.headers` though, not accessing the token directly. I'll add that to my to-do list. 

However, `Message` itself has access to the account so I'm not sure why you would be running into an issue there. Can you share the traceback you have, and the relevant code as well if possible?

MeridianExplorer commented:
Hey, I'm sorry for leaving it like that;
After 'fixing' the issue, I found out I didn't fix it, since, just like you said - OutlookAccount suddenly *did* have the attribute. How very strange.
I couldn't find the time to actually go further, so I closed the PR and deleted the branch, thinking it won't show, but I see it did after all :)

So, back to the issue at hand - 
My code looks like this:

`random_subject = string_generator(20)`
`random_body = string_generator(500)`
`account = OutlookAccount(obtain_access_token())`
`account.send_email(body=random_body, subject=random_subject, to=['user@email.com'])`

**NOTE: The issue happens both with `new_email().send()` and `send_email()`**

`string_generator` and `obtain_access_token` are two functions I wrote on my own, both are pretty self-explanatory :)
The string generator generates randomized strings and returns them, the token obtainer connects with Microsoft and returns a string containing the access token.

Upon running, I get the following traceback:

> Traceback (most recent call last):
  File "C:\Program Files\JetBrains\PyCharm 2017.3.2\helpers\pydev\pydevd.py", line 1668, in <module>
    main()
  File "C:\Program Files\JetBrains\PyCharm 2017.3.2\helpers\pydev\pydevd.py", line 1662, in main
    globals = debugger.run(setup['file'], None, None, is_module)
  File "C:\Program Files\JetBrains\PyCharm 2017.3.2\helpers\pydev\pydevd.py", line 1072, in run
    pydev_imports.execfile(file, globals, locals)  # execute the script
  File "<Path to my script>", line 10, in <module>
    account.send_email(body=random_body, subject=random_subject, to=['user@email.com'])
  File "C:\Python27\lib\site-packages\pyOutlook\core\main.py", line 214, in send_email
    email.send()
  File "C:\Python27\lib\site-packages\pyOutlook\core\message.py", line 310, in send
    self._make_api_call('post', endpoint=endpoint, data=json.dumps(payload))
  File "C:\Python27\lib\site-packages\pyOutlook\core\message.py", line 277, in _make_api_call
    headers = {"Authorization": "Bearer " + self.account.access_token, "Content-Type": "application/json"}
AttributeError: 'str' object has no attribute 'access_token'

Upon further inspection, this appears in the debugger
![image](https://user-images.githubusercontent.com/19227853/36717909-931a5cac-1ba8-11e8-8e2f-109beb55167d.png)
Which affirms what I used to think - account is simply a string containing the access token - not an `OutlookAccount ` object with an access_token attribute!

However, further down my code I actually forward a mail, using `forward()`.
It used to work just fine until I removed the attempt to go to `access_token`.
However - bam. Another crash. Now it sends emails, but doesn't forward them.

> Traceback (most recent call last):
  File "C:\Program Files\JetBrains\PyCharm 2017.3.2\helpers\pydev\pydevd.py", line 1668, in <module>
    main()
  File "C:\Program Files\JetBrains\PyCharm 2017.3.2\helpers\pydev\pydevd.py", line 1662, in main
    globals = debugger.run(setup['file'], None, None, is_module)
  File "C:\Program Files\JetBrains\PyCharm 2017.3.2\helpers\pydev\pydevd.py", line 1072, in run
    pydev_imports.execfile(file, globals, locals)  # execute the script
  File "<Path to my script>", line 15, in <module>
    inbox[0].forward(['otheremail@email.com'])
  File "C:\Python27\lib\site-packages\pyOutlook\core\message.py", line 343, in forward
    self._make_api_call('post', endpoint=endpoint, data=json.dumps(payload))
  File "C:\Python27\lib\site-packages\pyOutlook\core\message.py", line 277, in _make_api_call
    headers = {"Authorization": "Bearer " + self.account, "Content-Type": "application/json"}
TypeError: cannot concatenate 'str' and 'OutlookAccount' objects

This time, it actually is an object! With the correct attribute!
![image](https://user-images.githubusercontent.com/19227853/36718203-9cfefede-1ba9-11e8-8ac7-cafd3eb3004c.png)

Since I'm under heavy load, I wrote a workaround for it using a try-except block that tries both options, and it works perfectly.

>try:
    headers = {"Authorization": "Bearer " + self.account.access_token, "Content-Type": "application/json"}
except:
    headers = {"Authorization": "Bearer " + self.account, "Content-Type": "application/json"}

I know it's a horrid practice, but sadly I couldn't find the time to fix it correctly.
I hope the information helped :)
